src/pages/TodosPage.tsx

```diff
@@ -1,57 +1,45 @@
-import React, { useState, useEffect } from 'react';
+import React, {  useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import { TodoForm } from '../components/TodoForm';
 import { TodoList } from '../components/TodoList';
 import { ITodo } from '../interfaces';
-import { fetchTodos } from '../services/api';
+import { addTodo, toggleTodo, removeTodo, fetchTodosRequest } from '../actions';
+import {Action} from "redux-saga";
+import {ThunkDispatch} from "@reduxjs/toolkit";
+import {RootState} from "../store";
 
 declare var confirm: (question: string) => boolean;
```

Comment:
I would replace it with a new syntax: arrow function for example or even just remove it since there is only one place to use